requirements.txt

```diff
@@ -5,15 +5,15 @@
 #    pip-compile --output-file requirements.txt requirements.in
 #
 arrow==0.12.1
-backports.functools-lru-cache==1.4 ; python_version < "3.3"
+backports.functools-lru-cache==1.5 ; python_version < "3.3"
 click==6.7
 flask==0.12.2
 itsdangerous==0.24        # via flask
 jinja2==2.10              # via flask
 markupsafe==1.0           # via jinja2
 nose==1.3.7
-python-dateutil==2.6.1    # via arrow
+python-dateutil==2.7.0    # via arrow
```

Comment:
Please add constraint for this, since It causes python 2.6 test run to fail. Like

    python-dateutil==2.6.1 ; python_version < 2.7   # via arrow
    python-dateutil==2.7.0 ; python_version >= 2.7   # via arrow

Also It makes sense to constraint RQ the same way, since they have dropped Python 2.6 support.

Reply:
Do you mean like this? (See the added commit.)

I just thought that if 2.6 is no longer supported by RQ, nor by the Python developers, maybe rq-dashboard could also drop support for it.

Reply:
Yes. Thanks.
I decided to keep compatibility with 2.6 for now since I know some people who still use CentOS/RHEL 6 with default python which is 2.6. But I want to drop support for It in next major release.

Reply:
I see. I've squashed the commits into one.